packages/api/package.json

```diff
@@ -90,6 +91,7 @@
     "mocha": "^10.0.0",
     "mock-require": "^3.0.3",
     "nyc": "^15.1.0",
+    "oas-normalize": "^7.1.0",
```

Comment:
Loading this in to convert Swagger specs that we get back from https://apis.guru into OpenAPI specs that `api` can codegen from as `api` doesn't support Swagger right now (ever?).